config/locales/exceptions.en.yml

```diff
@@ -88,17 +88,34 @@ en:
         detail: "The record identified by %{id} could not be found"
         code: 404
         status: 404
+      resource_not_found:
+        <<: *defaults
+        title: Resource not found
+        detail: The requested resource could not be found
+        code: 404
```

Comment:
Why not reuse `record_not_found`?

Reply:
RecordNotFound requires an id, and sometimes there isn't a single id used for lookup (letters) or the id should not (or won't) be logged as in this case when the the id is the user's ssn. One option is to modify RecordNotFound so the id is optional, will also need to override the i18n detail from `errors` if the id is missing.